src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/CommonApiClient.java

```diff
@@ -33,10 +35,13 @@ public HttpResponse<String> sendAuthorizationRequest(String sessionId)
                         .setPath(this.clientConfigurationService.createUriPath("authorization"))
                         .addParameter(
                                 "redirect_uri",
-                                new URIBuilder(this.clientConfigurationService.getIPVCoreStubURL())
-                                        .setPath("/callback")
-                                        .build()
-                                        .toString())
+                                ssmHelper
+                                        .getParameterValueByName(
+                                                "/common-cri-api/clients/"
```

Comment:
just for added flexibility here as well use the same approach you've used, below so we use default of "common-cri-api" but can override through env variables

 this.clientId =
                Optional.ofNullable(System.getenv("COMMON_STACK"))
                        .orElse("common-cri-api");